src/components/ListItem.jsx

```diff
@@ -1,5 +1,76 @@
+import { useEffect, useState } from 'react';
+import { useToggle } from '@uidotdev/usehooks';
+import { Toggle } from './Toggle.jsx';
 import './ListItem.css';
+import { updateItem } from '../api/firebase.js';
+// import { getFutureDate} from '../utils/dates.js'
```

Comment:
Nit: To keep things tidy, I tend to delete commented out bits of code before merging. Not a blocker, but would be a good thing to go back and clean up if you have a sec.

Reply:
Thank you, @meganesu! We intentionally left it in to discuss our challenges with the getFutureDate function. We'll remove it for now and make sure to bring it up in today's discussion.